src/main/java/ViewServlet.java

```diff
@@ -0,0 +1,44 @@
+import lib.DatabaseTools;
+import lib.TransformationHelper;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.sql.Connection;
+
+@WebServlet("/view")
+public class ViewServlet extends HttpServlet {
+
+    private final DatabaseTools databaseTools = new DatabaseTools();
+    private final String query = "SELECT * FROM public.";
```

Comment:
```suggestion
    private final String query = "SELECT * FROM public.%s";
```
I niżej format %s dałbym. W obecnym rozwiąznaiu nie wiadomo czy to jest rozszerzane czy nie (mimo że można się domyślić z braku nazwy tabeli na końcu ale still).